packages/frontend-api/src/Resources/config/graphql-types/ModelType/Order/OrderDecorator.types.yaml

```
@@ -112,6 +112,9 @@ OrderDecorator:
             isPaid:
                 type: "Boolean!"
                 description: "Indicates whether the order is paid successfully with GoPay payment type"
+            isPaymentInProcess:
```

```suggestion
            hasPaymentInProcess:
```

Pls fix this in other occurences